README.md

```diff
@@ -157,13 +252,9 @@ Returns a Python dictionary of information about the client making an
 API request (such as its ip address, user, account,
```

Comment:
client has been with a capital C all through this docs until this instance. I think this should be Client too.